### ISSUE TITLE
Notebook user_acquisition.ipynb gives an error

### ISSUE THREAD

**Issue by miskamet** [superlinked/superlinked#67]
Notebooks example gives an AttributeError: 'NoneType' object has no attribute 'endswith'

I am using macbook
Reproducing: clone the repo and run through the notebook

**Comment by kembala** [comment]
Could you provide some more info on what environment did you use?
Based on the cloning i would assume you run something locally.

I will page @morkapronczay to help out once we have some more context available.

**Comment by miskamet** [comment]
Yes, I did use conda-created virtual environment with python 3.10.4
I have not installed anything on the environment, I did run the commands
```
%pip install superlinked==9.37.1
%pip install umap-learn
```
and needed ipykernel packages so it can be used as a notebook kernel, that's all.

The whole traceback:

<img width="1077" alt="Screenshot 2024-09-19 at 7 41 41" src="https://github.com/user-attachments/assets/311ec232-ee22-4375-bcd7-69dd3493d0a8">

**Comment by mamayer19** [comment]
Hi @miskamet,

Can you try with the latest superlinked `10.2.2`?

**Comment by kembala** [comment]
Unfortunately we cannot replicate the issue on our end.

**Comment by miskamet** [comment]
Yes, I noticed that the whole structure of the repo has changed after this bug. You can probably close the issue, I'll try to run the notebook again when I have time.
